test/Azure.Functions.Cli.Tests/ActionsTests/ResolveActionTests.cs

```diff
@@ -93,7 +93,8 @@ private IContainer InitializeContainerForTests()
                 .SingleInstance();
 
             builder.RegisterType<ProcessManager>()
-                .As<IProcessManager>();
+                .As<IProcessManager>()
+                .SingleInstance();
```

Comment:
This is a safe change because the [current implementation](https://github.com/Azure/azure-functions-core-tools/blob/6b9a1c051c83aaa33366a55463de6bb92631d24b/src/Azure.Functions.Cli/Common/ProcessManager.cs#L8) does not store any state other than the child process count). We want a single instance (which was used to register the child process) so that we have the same child processes when we handle the Ctrl + Break key event handler, where we terminate the child processes.